README.md

```diff
@@ -80,7 +80,21 @@ us before a training sesison.
 
 ## Installation and setup
 
-### 1. Clone or fork the repository
+There are two options for participating in this workshop for which instructions are provided below:
+
+* via a [local install](#local-install)
+* on [Google Colab](#google-colab)
```

Comment:
I think there's a mismatch between this link and the `#colab` subsection

Reply:
Thanks, good catch! Fixed.